src/core/lib/channel/channel_stack_builder.h

```diff
@@ -43,7 +43,7 @@ void grpc_channel_stack_builder_set_name(grpc_channel_stack_builder* builder,
 void grpc_channel_stack_builder_set_target(grpc_channel_stack_builder* b,
                                            const char* target);
 
-const char* grpc_channel_stack_builder_get_target(
+std::string grpc_channel_stack_builder_get_target(
```

Comment:
Would it make sense for this to return `absl::string_view` and have the caller convert to `std::string` if they need to save it?